src/sage/rings/lazy_series_ring.py

```diff
@@ -1765,6 +2083,14 @@ def _monomial(self, c, n):
         """
         return self._laurent_poly_ring(c).shift(n)
 
+    def _terms_of_degree(self, n, R):
+        r"""
+        Return the list consisting of a single element ``1`` in the given
+        ring.
+
+        """
```

Comment:
The coverage police very kindly asks for a doctest to be added here, or else.

Reply:
done, thank you!